src/Admin/AliasAdmin.php

```diff
@@ -98,6 +119,10 @@ protected function configureBatchActions($actions)
     public function prePersist($alias)
     {
         if (null == $alias->getDestination()) {
+            if (null == $alias->getUser()) {
+                # set user_id to current user if neither destination nor user_id is given
+                $alias->setUser($this->storage->getToken()->getUser());
```

Comment:
Here you should set the destination with the email from the user.

Reply:
That's excactly what I'm doing. But if no user is given, I'm getting the user first from the current admin. Afterwards the destination is still updated.

Reply:
Ah okay, its a bit tricky to unterstand.